src/CONFIGURATION.h

```diff
@@ -26,18 +37,11 @@ pins_t pins = pins_v0_5; // select the one that matches your board revision
 // pins_v0_1
 
 // Pusher Settings
-pusherType_t pusherType = PUSHER_SOLENOID_OPENLOOP; // either PUSHER_MOTOR_CLOSEDLOOP or PUSHER_SOLENOID_OPENLOOP
-uint16_t burstLength = 3;
-uint8_t bufferMode = 1;
-// 0 = stop firing when trigger is released
-// 1 = complete current burst when trigger is released
-// 2 = fire as many bursts as trigger pulls
-// for full auto, set burstLength high (50+) and bufferMode = 0
-uint16_t firingDelay_ms = 100; // delay to allow flywheels to spin up before firing dart
+pusherType_t pusherType = PUSHER_MOTOR_CLOSEDLOOP; // either PUSHER_MOTOR_CLOSEDLOOP or PUSHER_SOLENOID_OPENLOOP
```

Comment:
@ahalekelly We should probably have a discussion about how we handle defaults. For example, is JPEG going to become the poster Detlaff blaster, and thus should all the defaults should cater to it?

Reply:
Yeah I think catering the defaults to JPEG sounds reasonable